Update version number to 0.1.1.release

tshoang opened the pull request.

tshoang committed:
Removed unused Makefile

tshoang committed:
Update version number to 0.1.1.release

Also update README.md

tshoang committed:
Merge remote-tracking branch 'eBSoton@gh/master'

asiehsalehi reviewed (approved):
looks good to me, thanks

cfsnook commented:
looks like there is a problem in the build script?

tshoang committed:
FIxed the Travis configuration

Instead of running some non-existance make, run Maven

tshoang committed:
Event-B EMF Record Feature: Fixed Update Site Name

The Update Site name is now "Soton Plug-ins" to be consistent with the Rodin Platform
The version number is bumped to 0.1.2.qualifier

tshoang committed:
Event-B EMF Record Tests Feature: Fixed Update Site name

The Update Site name is now "Soton Plug-ins" to be consistent with the Rodin Platform
The version number is bumped to 0.1.2.qualifier

tshoang committed:
Event-B EMF Record SDK Feature: Fixed Update Site Name

The Update Site name is now "Soton Plug-ins" to be consistent with the Rodin Platform
The version number is bumped to 0.1.2.qualifier

tshoang committed:
All Features: Set version numbers to .release

tshoang commented:
This PR now contain update for 0.1.2.release

cfsnook commented:
thats better..
